src/borg/testsuite/helpers.py

```diff
@@ -14,39 +14,54 @@
 from ..archiver.prune_cmd import prune_within, prune_split
 from .. import platform
 from ..constants import MAX_DATA_SIZE
-from ..helpers import Location
-from ..helpers import Buffer
+from ..helpers import DEFAULTISH, FALSISH, SUPPORT_32BIT_PLATFORMS, TRUISH
 from ..helpers import (
-    partial_format,
+    Buffer,
+    ChunkerParams,
+    ChunkIteratorFileWrapper,
+    Location,
+    PlaceholderError,
+    ProgressIndicatorPercent,
+    StableDict,
+)
+from ..helpers import (
+    archivename_validator,
+    bin_to_hex,
+    binary_to_json,
+    chunkit,
+    clean_lines,
+    dash_open,
+    eval_escapes,
     format_file_size,
-    parse_file_size,
-    format_timedelta,
     format_line,
-    PlaceholderError,
+    format_timedelta,
+    get_base_dir,
+    get_cache_dir,
+    get_config_dir,
+    get_keys_dir,
+    get_runtime_dir,
+    get_security_dir,
+    interval,
+    is_slow_msgpack,
+    iter_separated,
+    make_path_safe,
+    msgpack,
+    parse_file_size,
+    parse_timestamp,
+    partial_format,
+    popen_with_error_handling,
+    remove_dotdot_prefixes,
     replace_placeholders,
+    safe_ns,
+    safe_s,
+    safe_unlink,
+    swidth_slice,
+    text_to_json,
+    text_validator,
+    yes,
 )
```

Comment:
That stuff uses to be grouped (more or less what belongs together somehow).

Or did you somehow trigger black doing this?

I rather prefer the horizontal grouped form, less scrolling.

Reply:
I'll get this changed back to the original 🙂